### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by larsnystrom** [iqbalfn/exec-php#14]
Thank you for exec-php, it's been very helpful.

I'm using it in my mocha tests, but I'm getting deprecation warnings both for `os.tmpDir() is deprecated. Use os.tmpdir() instead.` and  `Calling an asynchronous function without callback is deprecated.`.

Here are some stack traces:

```
(node:53568) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
    at makeCallback (fs.js:127:12)
    at Object.fs.unlink (fs.js:1049:14)
    at /path/to/my/app/node_modules/exec-php/lib/cli.js:53:20
    at ChildProcess.exithandler (child_process.js:267:7)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
```

```
(node:53568) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
    at Object.<anonymous> (/path/to/my/app/node_modules/exec-php/node_modules/tmp/lib/tmp.js:25:10)
    at Module._compile (module.js:643:30)
    at Module._extensions..js (module.js:654:10)
    at Object.require.extensions.(anonymous function) [as .js] 
```

**Comment by iqbalfn** [comment]
The `os.tmpDir` error is actually from other node_modules project, not sure if I've to take the module out and rewrite it manually.

While the async error may get fixed from [the latest merger request](https://github.com/iqbalfn/exec-php/pull/13) which is not merged yet do you failed on test.

Will fix it in the near future. Or if you've some time to fix, please create pull request.

**Comment by iqbalfn** [comment]
The issue already fixed on v0.0.4.

**Comment by larsnystrom** [comment]
Thank you!
